app/src/main/java/com/depromeet/team6/presentation/ui/home/HomeScreen.kt

```diff
@@ -21,32 +29,100 @@ fun HomeScreen(
     modifier: Modifier = Modifier,
     viewModel: HomeViewModel = hiltViewModel()
 ) {
+    val uiState = viewModel.uiState.collectAsState().value
```

Comment:
```suggestion
    val uiState by viewModel.uiState.collectAsStateWithLifecycle()
```

collectAsState는 Composable이 onStop이 되어도 데이터를 계속 수집하지만
collectAsStateWithLifecycle은 LifeCycle을 자동으로 관리해주기 때문에 안정적이고 성능도 좋아질거라고 생각해요!

Reply:
오 스터디 때 얘기했던 내용!